pkg/controller/certificates/groupapprove.go

```diff
@@ -72,7 +72,7 @@ func (cc *groupApprover) AutoApprove(csr *certificates.CertificateSigningRequest
 	if len(x509cr.DNSNames)+len(x509cr.EmailAddresses)+len(x509cr.IPAddresses) != 0 {
 		return csr, nil
 	}
-	if !hasExactUsages(csr, kubeletClientUsages) {
+	if !hasExactUsages(csr, kubeletClientUsages) && !hasExactUsages(csr, kubeletServerUsages) {
```

Comment:
It is my hope to put some bare minimum into this PR and move on to https://github.com/kubernetes/kubernetes/issues/45030 next, after getting some feedback on that issue. @mikedanese says he thinks it would be a relatively small change, but it will depend on feedback.

If it seems reasonable to proceed here with some bare minimum, would addressing @ericchiang's comment be enough?